.github/workflows/pull-request.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             
       - name: Build and Test
         run: |
-          xcodebuild -scheme GDSCommon-Package test -destination "platform=iOS Simulator,name=iPhone 14,OS=latest" \
+          xcodebuild -scheme GDSCommon-Demo test -destination "platform=iOS Simulator,name=iPhone 14,OS=latest" \
```

Comment:
@amopatel1234 Is this change needed, or do we need both here?

Reply:
if the `GDSCommon-Demo` scheme contains the tests for the package then no both are not needed, just demo is fine